packages/molecules/src/list/index.tsx

```
@@ -26,103 +77,95 @@ export const List: React.FC<ListType> = ({ items, label, noItem, onDelete, style
-                        )}
-                      </React.Fragment>
-                    }
-                  />
-                )}
-                {item?.items && <>{openItem === item?.id ? <ExpandLess /> : <ExpandMore />}</>}
-                <IconButton edge="end" aria-label="delete" onClick={() => handleDelete(item.id)}>
-                  <Delete />
-                </IconButton>
+                />
+              )}
+              {item?.items && <>{openItem === item?.id ? <ExpandLess /> : <ExpandMore />}</>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)